packages/sirius-web/backend/sirius-web-services/src/main/java/org/eclipse/sirius/web/services/documents/UploadDocumentEventHandler.java

```diff
@@ -85,18 +79,19 @@ public class UploadDocumentEventHandler implements IEditingContextEventHandler {
 
     private final IEditingContextMetadataProvider editingContextMetadataProvider;
 
+    private final List<IResourceLoaderService> resourceLoaderService;
```

Comment:
`org.eclipse.sirius.web.application.document.services.UploadDocumentEventHandler` needs to be updated too.

Reply:
Ok, I will add it also in the new `org.eclipse.sirius.web.application.document.services.UploadDocumentEventHandler`